torch/_export/logical_schema.py

```diff
@@ -206,16 +206,21 @@ class Tensor:
 
 
 ################################################################################
-# Following section is the defining the schema of 3 level construct: GraphModule, Graph, Node
+# Following section is defining the schema of 3 level construct: GraphModule, Graph, Node
 
-# IValue has no corresponding class in fx
-# IValue is the "values" that are passed between nodes in the graph
-# IValue is a named virtual tensor, with an optional TensorMeta that describes the properties of the tensor
-# !!! Consider using a more descriptive name, e.g. TensorValue, TensorPlaceholder, TensorArgument, etc.
+# TensorValue has no corresponding class in fx
+# TensorValue is the "tensor results" that are passed between nodes in the graph
+# TensorValue is a named virtual tensor, with an optional TensorMeta that describes the properties of the tensor
 @dataclass
-class IValue:
-    meta: TensorMeta
+class TensorValue:
```

Comment:
Logically, `TensorValue` and `SymIntValue` can  inherit from `Value`. 

But I am not sure if we want to introduce inheritance, so I didn't define a `Value` struct.